ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
The updated code in the PR that deals with this property does not seem to do anything other than ensure that the property com.arjuna.ats.arjuna.common.propertiesFile takes the value "commitmarkableresourcejbossts-properties.xml" and this can be achieved by setting the property without any tests.

It is good practice to clean up in tear down but that seems unrelated to my comment.